src/app/core/services/expense-fields.service.spec.ts

```
@@ -86,4 +91,12 @@ describe('ExpenseFieldsService', () => {
         done();
       });
   });
+
+  it('should return correct mapping for column name', () => {
+    expect(expenseFieldsService.getColumnName('spent_at')).toBe('txn_dt');
+    expect(expenseFieldsService.getColumnName('locations[0]')).toBe('location1');
+    expect(expenseFieldsService.getColumnName('travel_classes[0]', 2)).toBe('bus_travel_class');
+    expect(expenseFieldsService.getColumnName('travel_classes[1]', 1)).toBe('flight_return_travel_class');
+    expect(expenseFieldsService.getColumnName('random_string')).toBe('random_string');
```

Why are we handlign the random string case? Shouldnt we throw an error in these cases?

---

random_string  -> project
Add a comment that this is for supporting pre-existing normal cases like project, cost center etc

---

Done